templated_docs/templatetags/templated_docs_tags.py

```diff
@@ -6,39 +6,56 @@
 from django.utils.safestring import mark_safe
 from django.utils.html import escape
 from django import template
-register = template.Library()
-
-PIXEL_TO_CM = 0.00846666
 
+register = template.Library()
 
 class ImageNode(template.Node):
+
     def __init__(self, value):
-        self.value = template.Variable(value)
+        self.image_field = template.Variable(value)
+
+        try:
+            Img
+        except:
+            global Img
+            from PIL import Image as Img # User should not be forced to install PIL if he does not utilize images
+
 
     def render(self, context):
         try:
-            self.value = self.value.resolve(context)
+            self.value = self.image_field.resolve(context)
             if not isinstance(self.value, ImageFieldFile):
                 raise template.VariableDoesNotExist(
                     'Image argument should be an ImageField')
             images = context.dicts[0].setdefault('ootemplate_imgs', {})
-
             id = len(images)
             z_index = id + 3  # Magic
-            width = self.value.width * PIXEL_TO_CM
-            height = self.value.height * PIXEL_TO_CM
+            x_dpi = Img.open(self.value.path).info['dpi'][0]
```

Comment:
I'm not a specialist in image formats, but this works for `jpg` as `dpi` provided in `exif`. Cannot say for other image formats. For `png` this approach won't work.

So ether `dpi/ppi` to be frozen (which is imho weird), or currently allow limited image formats.
In fact, GIMP seem to recognize `ppi` for different formats. But currently I have no idea *how* to implement such a functionality in project in question.

Obviously, `pillow` is required on Django's side, therefore, available.